### PR TITLE
Beer Store

### LINKED ISSUE #128: The Beer Store
http://www.thebeerstore.ca/locations

### PR DESCRIPTION
**Pull request by sweetdream0729**
fixed #128

### CODE REVIEW

File: locations/spiders/beer_store.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import re
+
+from locations.items import GeojsonPointItem
+
+
+class BearStoreSpider(scrapy.Spider):
```
Comment (iandees) [inline locations/spiders/beer_store.py]: `BeerStore`, not `BearStore`.

File: locations/spiders/beer_store.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    name = "bearstore"
```
Comment (iandees) [inline locations/spiders/beer_store.py]: `beer_store`, not `bearstore`.

File: locations/spiders/beer_store.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    def parse(self, response):
+        results = json.loads(response.body_as_unicode())
+        features = results["features"]
+        for data in features:
+            description = data['properties']['description']
+            end_str = "<a href"
+            properties = {
+                'ref': data['properties']['storeid'],
+                'lon': data['geometry']['coordinates'][1],
```
Comment (iandees) [inline locations/spiders/beer_store.py]: Lat and Lon are flipped.

Comment (sweetdream0729) [reply locations/spiders/beer_store.py]: sorry, fixed typo and lat/lon

**Pull request merged**